Resources/Locale/en-US/reagents/meta/medicine.ftl

```diff
@@ -17,7 +17,7 @@ reagent-name-cryoxadone = cryoxadone
 reagent-desc-cryoxadone = Required for the proper function of cryogenics. Heals all standard types of damage, but only works in temperatures under 213K. It can treat and rejuvenate plants when applied in small doses.
 
 reagent-name-doxarubixadone = doxarubixadone
-reagent-desc-doxarubixadone = A cryogenics chemical. Heals certain types of cellular damage done by Slimes and improper use of other chemicals.
+reagent-desc-doxarubixadone = A cryogenics chemical. Heals certain types of cellular damage done by Slimes and improper use of other chemicals. Mainly heals cellular damage dealt by changelings. Works on dead people.
```

Comment:
i would not put changelings here as people dont know what a changeling is. or, this could also be a sign that we need more clarifications for changelings

Reply:
just so we're crystal clear, saying that it heals massive amounts of cellular damage is a good way to signal to people that this IS supposed to be used for lings